modules/manage/pages/manage-xdcr/incoming-xdcr-replications.adoc

```diff
@@ -0,0 +1,63 @@
+= Incoming Replications
+:description: pass:q[Incoming replications details for a target cluster in XDCR are listed on the UI and can be retrieved using the REST API.]
+:page-toclevels: 2
+
+[abstract]
+{description}
+
+When a replication is created, the target cluster is identified as such.
+
+A cluster is identified as a target cluster in XDCR when it receives replicated data from a remote cluster.
+Then the replicating remote cluster is identified as a source cluster.
+
```

Comment:
Add:
On the source cluster, when a replication is created, the replication is called an outgoing replication, and on the target cluster, the same replication is called an incoming replication.